curriculum/js1/4.js

```diff
@@ -7,7 +7,11 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+  const arr = []
+  for (let i = 0; i < a; i++) {
+    arr.push(b);
+  }
+  return arr.join("");
 };
```

Comment:
write this so that it's a recursive function; ALL problems should be answered recursively.